### DISCUSSION TITLE
News on the design system initiative

### DISCUSSION THREAD

**Discussion by ossamabenjemaa** [amnestywebsite/humanity-theme#476] [Enhancement Requests]
Hi there, @AmnestyAM @carolinejcourtney

Two months ago, we talked about a Figma file or design system for the Humanity WordPress theme. I’m writing to let you know that I’ve finally started working on it. Within our team, we use the theme, and the need for a design system became clear since we are a multidisciplinary team (product managers, designers, developers).

A design system would allow designers to work independently, especially those without knowledge of WordPress.

Initially, I looked into Amnesty’s guidelines and found this resource: https://brandhub.amnesty.org/visual-identity/. I then started mapping out my approach. My first step was to list all the blocks and patterns, then implement them one by one in WordPress. I’m using the inspector to identify font sizes, layouts, and other CSS elements to keep the blocks as accurate as possible.

If you’re interested, I’d be glad to share the product, which I’ve humbly named “Unity.”

I’d love to discuss this further; the current version (v1) is still in development and undergoing testing. I’d appreciate your feedback, especially on the accuracy of UI elements (font sizes, margins, etc.).

I admit I sometimes had difficulty identifying everything accurately, but I’m getting there.

That’s all for now!

Ossama from Fairness
cc Bertrand ( @volubyl ) - web developer fom Fairness

**Comment by ossamabenjemaa** [comment]
Up ?

**Comment by carolinejcourtney** [comment]
Hi Ossama - sorry for the delay. Yes, please share what you have, I'd love to see it. Will message you directly with my email addres :)
